src/Symfony/Component/Form/Button.php

```diff
@@ -156,7 +154,7 @@ public function get($name)
     /**
      * Unsupported method.
      *
-     * @param string $name
+     * @param string|int $name
      *
```

Comment:
Docblock changes should usually go to lower branches. Please take care of this when reviewing.

Reply:
This by the way looks like something that should happen anymore thanks to the changes to the `ButtonBuilder` in 4.3 IIRC.